### ISSUE TITLE
The hole in plugging task is impenetrable.

### ISSUE THREAD

**Issue by Director-of-G** [DavidPL1/assembly_example#18]
Hi, the plugging task aims to insert the plug into a hole. However, we found the hole is impenetrable, which indicates the object mesh is not loaded into Mujoco correctly. We say impenetrable here because objects which are thinner than the hole cannot be inserted into it.

![bug_report](https://user-images.githubusercontent.com/73640657/230768558-c1542744-062b-4d9e-b832-a18ba7bbd75d.png)

An example of such case is provided in [Google Drive](https://drive.google.com/drive/folders/1ev2U2ivAhkAkT4VxSsNe1A5BmfIWJYu8?usp=share_link), we ran the test with docker.assembly_server-v2.1.tar.gz, the latest server image. We align the gripper with the object frame, and control the gripper to descend gradually until a large change in the sensed force is detected, which means the gripper clashes into obstacles. We recorded two topics, `/cameras/gripper_cam/rgb` and `/cameras/workspace_cam2/rgb` in [Google Drive](https://drive.google.com/drive/folders/1ev2U2ivAhkAkT4VxSsNe1A5BmfIWJYu8?usp=share_link), it is observed that contact is detected once the gripper comes into contact with the upper surface of the hole. We only tested with round holes.

Please check the simulation environment, as this possible bug hinders us from further development. Besides, we observed that the plugging scene was not loaded correctly at times (e.g., the plug fell onto the ground).Unfortunately, we failed to record an video to showcase such exceptions.

**Comment by Director-of-G** [comment]
We ran some more tests in docker.assembly_server-v2.1.tar.gz. We found that only the round hole is impenetrable, while the triangle and rectangle ones are normal. However, only round holes are provided in difficulty `1`, moreover, we could not manually decide the hole shapes in difficulty `2~4`. We hope the organizers @fpatzelt @DavidPL1 @balandbal  could update the server images as soon as possible, because this issue greatly prevents us from advancing and may also be affecting other participants.

We also observed cases where the plugs were not initially placed on the table, some fell onto the ground and some were quite far away. Although this does not occur often, we want to make sure such cases won't appear in final competition. Because the plugs may be out of Franka's workspace.

Thanks a lot!

**Comment by DavidPL1** [comment]
@balandbal is in charge of the plugging task and will fix this as soon as possible.
Here in Germany (like last Friday) it is a holiday today, so he probably won't start working on a fix before tomorrow.

**Comment by balandbal** [comment]
It's curious to hear that the cylindrical hole is still impenetrable; [version 2.1](https://github.com/DavidPL1/assembly_example/discussions/13) should have fixed just that.

In the next few days, I will look into the problem above, the parts dropping to the ground, and add the missing feature of providing the geometry data. I am sorry for any inconvenience.

**Comment by Director-of-G** [comment]
> It's curious to hear that the cylindrical hole is still impenetrable; https://github.com/DavidPL1/assembly_example/discussions/13 should have fixed just that.

Thanks for your reply. We downloaded again [version 2.1](https://github.com/DavidPL1/assembly_example/discussions/13) and the problem was gone. Actually, we noticed that the current image `s4dx/assembly_server:2.1` and the previous one we used was not exactly the same. They were reported to be `CREATED` on different days when we ran `docker images`. Maybe we have downloaded the old image before the launch was not announced officially.
Anyway, the impenetrable mesh problem is solved. Thanks again!
